style.css

```diff
@@ -79,4 +161,108 @@
     padding: 0;
     margin: 0;
   }
+
+  .works {
+    display: grid;
+    grid-template-columns: auto;
+    align-content: center;
+  }
+
+  .portfolio {
+    display: flex;
+    flex-direction: column;
+    margin: 0 24px 100px 24px;
+    padding: 16px;
+    background-color: white;
+    border-radius: 10px;
+  }
+
+  .about-me {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    padding: 24px;
+    margin: 0;
+    border: 0;
+    background-color: white;
+    border-radius: 0 60px 0 0;
+  }
+
+  .about {
+    margin: 100px 0;
+  }
+
+  .resume {
+    font-family: "Poppins", sans-serif;
+    font-size: 17px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    background-color: transparent;
+    padding: 15px;
+    border-radius: 8px;
+    border: 1px solid;
+    margin: 12px 0;
+  }
+
+  .resume:hover {
+    background-color: #6070ff;
+    color: white;
+  }
+
+  .resume:active {
+    background-color: #2230d2;
+    color: white;
+  }
+
+  .resume:disabled {
+    border: 1px solid;
+    color: #5e6c84;
+  }
+
+  h6 {
+    font-family: "Poppins", sans-serif;
+    font-variant-caps: titling-caps;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #000;
+    padding: 12px;
+    margin: 15px 0;
+  }
+
+  .technologies {
+    background-color: #f7f7f9;
+    border-radius: 8px;
+    list-style: none;
+    align-items: left;
+    padding: 0 12px;
+  }
```

Comment:
- [ ] Good work building your Languages section. ✔️ However, please kindly ensure to change the styles as requested by the previous reviewer. 🤓 Currently, I can still see significant differences between your project and the Figma design. 🔧

Figma design. 👇🏽
![f](https://github.com/firepicaso/RWD_Portfolio/assets/116701485/1facaf71-a356-446b-b7c6-23ec25723298)


Your design. 👇🏽
![d](https://github.com/firepicaso/RWD_Portfolio/assets/116701485/c4115b9d-af3d-4a56-9514-397d795cc456)